DEPLOYMENT_CHECKLIST.md

```diff
@@ -0,0 +1,80 @@
+- [ ] Check if there is an official ENS deployment for the choosen chain and if yes:
+  - [ ] Check if there is already an entry for it in `packages/contracts/deploy/helpers.ts`
+  - [ ] Check that the owner of the DAO domain is the deployer
+  - [ ] Check that the owner of the plugin domain is the deployer
+- [ ] Set `PRE_DEPLOYMENT_TESTS` to `true` in `.env` to enable warnings from proxy tests
+- [ ] Run `yarn` in the repository root to install the dependencies
+- [ ] Run `yarn build` in `packages/contracts` to make sure the contracts compile
+- [ ] Run `yarn test` in `packages/contracts` to make sure the contract tests succeed
+- [ ] Run `yarn deploy --network hardhat --reset` to make sure the deploy scripts work
+- [ ] Set `ETH_KEY` in `.env` to the deployers private key
+- [ ] Set the right API key for the chains blockchain explorer in `.env` (e.g. for mainnet it is `ETHERSCAN_KEY`)
+- [ ] Set the choosen DAO ENS domain to `NETWORK_DAO_ENS_DOMAIN` in `.env` and replace `NETWORK` with the correct network name (e.g. for mainnet it is `MAINNET_DAO_ENS_DOMAIN`)
+- [ ] Set the choosen Plugin ENS domain to `NETWORK_PLUGIN_ENS_DOMAIN` in `.env` and replace `NETWORK` with the correct network name (e.g. for mainnet it is `MAINNET_PLUGIN_ENS_DOMAIN`)
+
+## Deployment
+
+To deploy run `yarn deploy --network NETWORK` in `packages/contracts` and replace `NETWORK` with the correct network name (e.g. for mainnet it is `yarn deploy --network NETWORK`)
+
+## After-Deployment
+
+### Verification
+
+- [ ] Take the addresses from this file `packages/contracts/deployed_contracts.json`
+- [ ] Wait for the deployment script finishing verification
+- [ ] Go to the blockchain explorer and verify that each address is verified
+  - [ ] If it is not try to verfiy it with `npx hardhat verify --network NETWORK ADDRESS CONTRUCTOR-ARGS`. More infos on how to use this command can be found here: [https://hardhat.org/hardhat-runner/docs/guides/verifying](https://hardhat.org/hardhat-runner/docs/guides/verifying)
+  - [ ] If it is a proxy try to activate the blockchain explorer' proxy feature
+- [ ] Update `active_contracts.json` with the new deployed addresses
+- [ ] Update `packages/contracts/Releases.md` with the new deployed addresses
+
+### Configurations
+
+- [ ] Check if the managing DAO set in the `DAO_ENSSubdomainRegistrar`
+- [ ] Check if the managing DAO set in the `Plugin_ENSSubdomainRegistrar`
+- [ ] Check if the managing DAO set in the `DAORegistry`
+- [ ] Check if the managing DAO set in the `PluginRepoRegistry`
+- [ ] Check if the `PluginRepoRegistry` is set in the `PluginRepoFactory`
+- [ ] Check if the managing DAO set in the `PluginSetupProcessor`
+- [ ] Check if the `PluginRepoRegistry` set in the `PluginSetupProcessor`
+- [ ] Check if the `DAORegistry` set in the `DAOFactory`
+- [ ] Check if the `PluginSetupProcessor` set in the `DAOFactory`
+
+### Permissions
+
+- [ ] Check that the deployer has the ROOT permission on the managing DAO
```

Comment:
We could listen on the registry because we could register our managing dao in the registry after the framework deployment. Another solution would be to hardcode it into the subgraph, the managing DAO will probably never change once the framework is deployed.

I created a task here to track this https://aragonassociation.atlassian.net/browse/APP-1695